authors/filip-srdoc/15-porezi/hr_HR/final.md

```diff
@@ -0,0 +1,107 @@
+Prema [FATF-u][fatf] (Financial Action Task Force) virtualni novac obilježavaju klasične funkcije novca koje kažu da je novac, pa tako i onaj virtualni, sredstvo razmijene, mjera vrijednosti te služi za pohranu vrijednosti. No, postoji razlika između klasičnog [fiat][fiat] novca i virtualnog novca. Naime, fiat novac je službeno sredstvo plaćanja u nekoj državi kojeg izdaje centralna banka te države za razliku od virtualnog novca koji nigdje nije prihvaćen kao službeno sredstvo plaćanja i koji nastaje rudarenjem o kojem više možete doznati [ovdje][bc].
```

Comment:
- ... novca: on je sredstvo **razmjene** te mjera i pohrana vrijednosti.
- banka te države, za razliku
- koji nastaje [rudarenjem ili stake-anjem](https://bitfalls.com/2017/10/23/whats-the-difference-between-proof-of-work-pow-proof-of-stake-pos-and-delegated-pos/)

Postoji razlika virtualni / digitalni novac. Virtualni se smatra "ne pravim", dok je digitalni pravi ali digitalizirani. Možda je bolje nazivati kriptovalute digitalnim novcem, ako ih već ne zovemo kriptovalutama direktno u uvodu.

Reply:
Virtualni novac je izmišeljeni novac koji isto tako ima obilježja tradiconalnog novca. To su sredstvo razmjene, mjera vrijednosti i mogučnost pohrane vrijednosti.
Tradicionalni novac izdaje isklučivo centralna odnsno središnja banka neke države.
Razlika između virtualnog i digitalnog novca je ta što je digitalni novac ubiti elektornička odnosno digitalna inačica fiat novca, a s obzirom da je i virutalni novac također u elektorničkom izdanju onda i on kao i fiat novac spada pod pojam digitalnog novca. Kriptovaluta i virtualna valuta su sinonimi tako da bi mozda bolje bilo da bi mozda bolje bilo da napisem tipa " najpoznatija virutalna tj. kriptovaluta je bitcoin". Virtualne valute se rudare. virtualne valute su u digitalnom(elektornickom) obliku. fiat valute se stampaju. elektornicka inacica fiat valute je takoder digitalna valuta

Reply:
jedino centralna ili sredisnja banka moze izdat novac zato sam to napisao 
kak ti zvuci ovo objasnjenje razlika koje sam sad napisao